test/backend/abc.in.cpp

```diff
@@ -21,6 +21,11 @@
 #include "util/test_control.hpp"
 #include "util/test_tools.hpp"
 
+#ifdef NGRAPH_UNIT_TEST_OPENVINO_ENABLE
```

Comment:
Instead of putting these lines in 100+ files, add it to one of the already included `util/` include files that the tests all include, along with a comment explaining why it's needed.

Reply:
Done, thanks! I added it to util/test_control.hpp